apps/condo/domains/organization/schema/Organization.js

```diff
@@ -156,6 +156,20 @@ const Organization = new GQLListSchema('Organization', {
             kmigratorOptions: { null: true, unique: false },
         },
         features: ORGANIZATION_FEATURES_FIELD,
+
+        isApproved: {
+            // NOTE: This field is used for analytics, you should not implement logic using it
+            // By default, all organizations are set to true.
+            // If the organization is test or fraudulent, then you need to set value to false.
+            schemaDoc: '@deprecated' +
```

Comment:
Born deprecated? 😆

Reply:
in order to scare away particularly persistent developers who want to use this field :)